### PR TITLE
Show ETH value input for payable methods (fix for solidity v0.6x)

### PR DESCRIPTION
**Pull request by nicosampler**
Partially Closes https://github.com/gnosis/safe-react/issues/1680

### CODE REVIEW

File: apps/tx-builder/src/components/Dashboard.tsx [hunk 1]
```diff
@@ -140,7 +140,11 @@ const Dashboard = () => {
 
   const getContractMethod = () => contract?.methods[selectedMethodIndex];
 
-  const isValueInputVisible = () => getContractMethod()?.payable;
+  const isValueInputVisible = () => {
+    const method = getContractMethod();
+    debugger
```
Comment (dasanra) [inline apps/tx-builder/src/components/Dashboard.tsx:145]: I think this is a leftover

Comment (nicosampler) [reply apps/tx-builder/src/components/Dashboard.tsx]: :open_mouth:

**Review by dasanra — APPROVED** [review]

**Comment by dasanra** [comment]
This app can be tested here

https://ipfs.io/ipfs/QmYES1Se6i6679z3PfQ62bydgVVEoSRUabvjB35DfUGPGA/

**Comment by francovenica** [comment]
Tested this in the https://github.com/gnosis/safe-react/pull/1717, using the URL given in Dani's comment

It looks good to me

**Pull request merged**